aws/ec2.go

```diff
@@ -314,6 +314,8 @@ func findSecurityGroupWithClusterID(svc ec2iface.EC2API, clusterID string, contr
 	}
 
 	if len(resp.SecurityGroups) < 1 {
+		log.Errorf("Cannot find security group that matches: %v\n", params)
+
 		return nil, ErrMissingSecurityGroup
```

Comment:
This error is already logged as `Fatal`. I suggest we just enhance this error to what you put above if it's not clear enough as is. otherwise we log related errors twice.

Reply:
I was thinking the same, but how to do that? Since the error is a typed var. Could I just return a new error with this formatted in?

Reply:
> Could I just return a new error with this formatted in?

Yes, we only use the var in this place so we can just drop it and return a formatted error.

Reply:
Fixed, the error will now look like this:
```
could not find security group that matches: [{
  Name: "tag-key",
  Values: ["kubernetes.io/cluster/foo"]
} {
  Name: "tag-value",
  Values: ["owned"]
} {
  Name: "tag-key",
  Values: ["kubernetes:application"]
} {
  Name: "tag-value",
  Values: ["kube-ingress-aws-controller"]
}]
```